quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/WidgetController.java

```diff
@@ -22,23 +25,25 @@
 @RequestMapping("/api/widgets")
 public class WidgetController {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)